frontend/src/app/modules/dashboard/views/chat/components/chat-text-box/chat-text-box.component.scss

```diff
@@ -1,10 +1,12 @@
+@import 'colors';
+
 :host {
   padding-left: 44px;
   margin-right: 32px;
 }
 
 .chat-text-box {
-  background: #eff2f4;
+  background: $chat-bg-color;
```

Comment:
Daca sa zicem ca e o exceptie de culoare, gen ai pus-o specific `$chat-bg-color` atunci pe nimeni nu mai intereseaza si ar trebui sa fie sus aici sub importuri variabila declarata , nu in _colors.scss

Reply:
fixed